infrastructure/src/main/java/org/corfudb/infrastructure/health/Component.java

```diff
@@ -0,0 +1,37 @@
+package org.corfudb.infrastructure.health;
+import com.google.gson.annotations.SerializedName;
```

Comment:
Any reason we can't use protobufs instead of gson. While it is probably ok for the current set of usecases, we have a lot of state represented in protobufs which we will eventually want to push to the health reports too and making this new layer protobuf based will avoid having a protobuf->gson translation layer later right?

Reply:
For now, it's the requirement that the report is in JSON